faster_raster.go

```diff
@@ -253,7 +253,6 @@ func (r *Rasterizer) Stop() {
 	// Send the quit signal to the mainEventLoop goroutine for this Rasterizer
 	if r.quitChan != nil {
 		close(r.quitChan)
-		r.quitChan = nil
```

Comment:
I think we could end up with the `mainEventLoop` never stopping if this statement gets executed before `case <-r.quitChan`. WDYT? Also, the race detector is not happy about setting `quitChan` to nil here.

Reply:
Yeah, despite setting channels to nil being quite useful, the race detector always thinks it's an unprotected access. In this case it comes immediately after the close and we don't even dynamically re-open the channel.